README.md

```diff
@@ -81,6 +113,21 @@ Type IV is supported if built-in methods consult properties on instances instead
 
 Note that `RegExp`'s @@match, @@matchAll, @@replace, @@search, and @@split symbols themselves aren't strictly only for subclassing support, as they are not used in `RegExp` methods themselves. Instead, they are used as a protocol for `String` so that completely custom `RegExp` instances may be consumed.
 
+### Example
+
+```js
+class R extends RegExp {
```

Comment:
This example should probably contain an empty constructor that does not call `super()`.

Reply:
+1